daft/expressions/expressions.py

```diff
@@ -466,6 +466,16 @@ def __invert__(self) -> Expression:
         expr = self._expr.__invert__()
         return Expression._from_pyexpr(expr)
 
+    def __floordiv__(self, other: Expression) -> Expression:
+        """Floor divides two numeric expressions (``e1 / e2``)"""
+        expr = Expression._to_expression(other)
+        return Expression._from_pyexpr(self._expr // expr._expr)
+
+    def __rfloordiv__(self, other: object) -> Expression:
```

Comment:
Is this used / tested anywhere?

Reply:
This follows the `__truediv__` and `__rtruediv__`. Currently only used in UT.